mobile/src/libs/packages/store/middlewares/middlewares.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  type AnyAction,
+  type Dispatch,
+  isRejected,
+  type Middleware,
+} from '@reduxjs/toolkit';
+
+import { NotificationType } from '#libs/enums/enums';
+import { type AsyncThunkConfig } from '#libs/types/types';
+import { appActions } from '#slices/app/notifications';
+
+import { type AppDispatch } from '../libs/types/types';
+
+const handleError: Middleware<AsyncThunkConfig, unknown, AppDispatch> = ({
```

Comment:
move it to separate file - handle-error.middleware.ts and re-export from middlewares.ts